sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+        """Add selected lots to the existing sale order."""
+        self.ensure_one()
+        if not self.sale_order_id:
+            raise ValidationError(_("Please select a Sale Order"))
+        self.sale_order_id.write(
+            {
+                "order_line": self._get_vals_for_add_lot_in_sale_order_line(),
+            }
+        )
+        return self._action_open_sale_order()
+
+    def action_create_sale_order(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `action_create_sale_order`

(ANN201)